### ISSUE TITLE
Innate Spellcasting not consuming charges

### ISSUE THREAD

**Issue by bushybushi** [jbhaywood/5e-statblock-importer#130]
Whenever you import a character with innate spellcasting, whenever you use a spell, the charges are not being used, I usually have to do it manually for each spell, I hope this can be resolved or added if it was not intended.

**Comment by Aioros** [comment]
Hey, thanks for the report, I'll look into it. Do you mind sharing the statblock you are using? Also the dnd5e system version would be useful.

**Comment by bushybushi** [comment]
Sure! I use DnD 5e version 4.1.2

and here is the statblock import I used:
```
Thryn
Large Fey, Neutral Good

Armor Class 20 (natural armor)
Hit Points 120 (13d10 + 52)
Speed 40 ft.

STR	DEX	CON	INT	WIS	CHA
20 (+5)	12 (+1)	18 (+4)	10 (+0)	14 (+2)	12 (+1)
Saving Throws Str +8, Con +7
Skills Athletics +8, Intimidation +4, Perception +5
Damage Resistances Bludgeoning, Piercing, and Slashing
Condition Immunities Frightened
Senses Darkvision 60 ft., Passive Perception 15
Languages Common, Sylvan, Telepathy 60 ft.
Challenge 8 (3,900 XP)

Fey Ancestry. Thryn has advantage on saving throws against being charmed, and magic can’t put him to sleep.

Keen Hearing and Smell. Thryn has advantage on Wisdom (Perception) checks that rely on hearing or smell.

Stone’s Resilience (1/Day). Thryn hardens his form with magical energy, gaining 20 temporary hit points. For 10 minutes, he has resistance to all damage except psychic.

Immovable Protector. Thryn has advantage on saving throws to resist being knocked prone or pushed.

Taunting Presence. As a bonus action, Thryn can target one creature within 30 feet that can see and hear him. The target must succeed on a DC 15 Wisdom saving throw or have disadvantage on attack rolls against any creature other than Thryn until the end of its next turn.

Guardian’s Aura. Allies within 10 feet of Thryn gain a +2 bonus to AC as long as he is conscious and not incapacitated.

Innate Spellcasting. Thryn’s innate spellcasting ability is Wisdom (spell save DC 13). He can innately cast the following spells, requiring no material components:

At will: Mold Earth, Thunderclap
2/day each: Misty Step, Shield
Actions
Multiattack. Thryn makes two attacks: one with his bite and one with his claws.

Bite. Melee Weapon Attack: +8 to hit, reach 5 ft., one target.
Hit: 10 (1d10 + 5) piercing damage.

Claws. Melee Weapon Attack: +8 to hit, reach 5 ft., one target.
Hit: 12 (2d6 + 5) slashing damage.

Legendary Actions
Thryn can take 2 legendary actions, choosing from the options below. Only one legendary action can be used at a time and only at the end of another creature's turn. Thryn regains spent legendary actions at the start of his turn.

Detect. Thryn makes a Wisdom (Perception) check.
Taunting Attack (Costs 2 Actions). Thryn targets one creature within 5 feet of him. The target must succeed on a DC 15 Wisdom saving throw or use its reaction to attempt to attack Thryn with disadvantage.
```

**Comment by Aioros** [comment]
Alright, it should be fixed now in v1.92.2. Let me know if it works for you after updating the module, and thanks again for the heads up.

**Comment by bushybushi** [comment]
Works perfectly! Thanks so much, you're a giga chad 🗿. Doing wonders for this community,
